bundles/org.openhab.binding.caddx/src/main/java/org/openhab/binding/caddx/internal/handler/ThingHandlerZone.java

```diff
@@ -52,7 +52,7 @@ public ThingHandlerZone(Thing thing) {
     @Override
     public void updateChannel(ChannelUID channelUID, String data) {
         if (channelUID.getId().equals(CaddxBindingConstants.ZONE_NAME)) {
-            getThing().setLabel(data);
+            // getThing().setLabel(data);
```

Comment:
Is this change intentional?

Reply:
Yes. The zone name is also exposed as a channel that can be used.

Reply:
Can the comment be removed, then?

Reply:
Done